app/components/blacklight/name_roles_facet_hierarchy_component.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Blacklight
```

Comment:
I'm not sure this should be name-spaced under `module Blacklight`. It makes it seem like an override when it's a local class that inherits behavior from `Blacklight::FacetFieldListComponent`.

Reply:
True, thanks. I was over-exuberant borrowing from https://github.com/sul-dlss/earthworks/blob/main/app/components/blacklight/date_facet_hierarchy_component.rb